packages/stryker-karma-runner/test/unit/KarmaTestRunnerSpec.ts

```diff
@@ -176,11 +170,6 @@ describe('KarmaTestRunner', () => {
       karmaRunStub.callsArgOn(1, 0);
     });
 
-    it('should execute "karma run"', async () => {
-      await sut.run({});
-      expect(karmaRunStub).calledWith({ port: settings.port });
```

Comment:
This test is still valid, just no longer with the port configuration

Reply:
it says "Property 'port' does not exist on type 'RunnerOptions'" in `settings.port`

Reply:
+ I think it is the same as that one from below